pipeline/Dockerfile.test

```diff
@@ -0,0 +1,20 @@
+FROM {{ base_image.image }}
+
+COPY hailjwt/setup.py /hailjwt/
+COPY hailjwt/hailjwt /hailjwt/hailjwt/
+RUN pip3 install --no-cache-dir /hailjwt && \
+  rm -rf /hailjwt
+
+COPY batch/setup.py /batch/
+COPY batch/batch/ /batch/batch/
+RUN pip3 install --no-cache-dir /batch && \
+  rm -rf /batch
+
+COPY pipeline/setup.py /pipeline/
+COPY pipeline/pipeline/ /pipeline/pipeline/
+RUN pip3 install --no-cache-dir /pipeline && \
```

Comment:
This confuses me. We're copying contents of `pipeline/pipeline to /pipeline/pipeline/`. However, in the next step we rm -rf /pipeline. Isn't the net effect that this copy step does nothing?